rmc/utils/constraint.py

```diff
@@ -1173,6 +1151,7 @@ def get_oe_annotation(ht: hl.Table, freeze: int) -> hl.Table:
     Use regional OE value if available, otherwise use transcript OE value.
 
     .. note::
+        - Assumes `constraint_prep` Table is missense-specific
```

Comment:
Hmm there actually isn't an easy way to check this as is with joins/variant annotations since this is a locus-level table, unless we do something like comparing the total obs or exp per transcript to what we expect from missense only. I wonder if we should actually write out the constraint prep Tables for diff variant consequences to diff paths?

Reply:
hmmm. I guess that would make this check easier, otherwise we'd have to aggregate on worst consequence and do the check that way. I'm fine with writing the tables to different paths if we add a note to delete them later